ansible_collections/arista/avd/docs/contribution/development-tooling.md

```diff
@@ -0,0 +1,74 @@
+<!--
+  ~ Copyright (c) 2023-2024 Arista Networks, Inc.
+  ~ Use of this source code is governed by the Apache License 2.0
+  ~ that can be found in the LICENSE file.
+  -->
+
+# Development Tooling
+
+## Makefile
+<!-- TO DO - update!!! -->
+
+## Pre-commit hook
+
+[pre-commit](https://github.com/aristanetworks/avd/blob/devel/.pre-commit-config.yaml) can run standard hooks on every commit to automatically point out issues in code such as missing semicolons, trailing whitespace, and debug statements. Pointing these issues out before code review allows a code reviewer to focus on the architecture of a change while not wasting time with trivial style nitpicks.
+
```

Comment:
The CI will perform the same checks which must be passed, so it's better for you if you run those tests and fix issues before submitting any changes.

Reply:
see commit 9b2bc1055ae33674b4288c2d640628ed4cc0ee4d